fidesctl/src/fidesctl/core/export_helpers.py

```diff
@@ -131,34 +137,86 @@ def get_formatted_data_use(
     """
     This function retrieves the data use from the server
     and formats the results, returning the necessary values
-    as a dict
+    as a dict. With multiple types possible, some introspection
+    is done to handle each accordingly.
     """
 
     data_use = get_server_resource(url, "data_use", data_use_fides_key, headers)
 
     formatted_data_use = {
         "name": data_use.name,
-        "legal_basis": "N/A",
-        "special_category": "N/A",
-        "recipients": "N/A",
     }
 
-    try:
-        formatted_data_use["legal_basis"] = data_use.legal_basis.value
-    except AttributeError:
-        echo_red("Legal Basis undefined for specified Data Use, setting as N/A.")
+    for attribute in [
+        "legal_basis",
+        "special_category",
+        "recipients",
+        "legitimate_interest_impact_assessment",
+        "legitimate_interest",
+    ]:
+        try:
+            attribute_value = getattr(data_use, attribute)
+            if attribute_value is None:
+                attribute_value = "N/A"
+            elif isinstance(attribute_value, Enum):
+                attribute_value = attribute_value.value
+            elif isinstance(attribute_value, list):
+                attribute_value = ", ".join(attribute_value)
+            elif attribute == "legitimate_interest":
+                if attribute_value is True:
+                    attribute_value = getattr(data_use, "name")
+                else:
+                    attribute_value = "N/A"
```

Comment:
Maybe this could be handled on the model instead? It may be a problem when applying the model but probably better to see as part of that workflow earlier versus when trying to see it. Alternatively, look at removing the validation altogether for maximum flexibility (can be something that is done as part of the compliance in #372 )

Reply:
This may make it possible to have a single function that can format both `data_use` and `data_subject`

Reply:
Less critical at this point in time so deferring for a later enhancement (looking at simplifying manifests in the future as well that could help here)